beacon-chain/types/crystallized_state.go

```diff
@@ -290,15 +289,20 @@ func (c *CrystallizedState) NewStateRecalculations(aState *ActiveState, block *B
 		lastStateRecalculationSlotCycleBack = c.LastStateRecalculationSlot() - params.GetConfig().CycleLength
 	}
 
+	blockVoteCache, err := db.ReadBlockVoteCache(recentBlockHashes[0:params.GetConfig().CycleLength])
```

Comment:
Can you please add a todo here? since clean up is not supported, `recentBlockHashes[0:params.GetConfig().CycleLength]` is most likely wrong. We can address that in the next PR

Reply:
711 is the issue number

Reply:
Sure. Added.